src/main/java/frc/robot/subsystems/arm/commands/SetElbowAngle.java

```diff
@@ -21,6 +21,6 @@ public void initialize() {
 
     @Override
     public boolean isFinished() {
-        return MathUtil.applyDeadband(angle - arm.getElbowJointAngle(), ArmConstants.SETPOINT_DEADBAND) == 0;
+        return MathUtil.applyDeadband(Math.toRadians(angle) - arm.getElbowJointAngle(), ArmConstants.SETPOINT_DEADBAND) == 0;
```

Comment:
Why not use degrees everywhere?

Reply:
All of the feed farward calculations use radians and IMO to ask the user for degrees and convert them